haiku-apps/wakeup/wakeup-1.0.recipe

```diff
@@ -0,0 +1,45 @@
+
+SUMMARY="An application that plays sound at given interval"
+DESCRIPTION="WakeUp v1.0,\
+Copyright (c)1999. WakeUp is a simple application that will play a given sound at a specified interval.\
+ Compiled under DeOS R4.5.1 (Intel Only)"
+HOMEPAGE="https://github.com/HaikuArchives/WakeUp"
+COPYRIGHT="1999  Jonathan Villemure"
+LICENSE="MIT"
+REVISION="1"
+
+SOURCE_URI="https://github.com/HaikuArchives/WakeUp/archive/116afedea73ba9779d3d641028b65fdb2dc19012.tar.gz"
+CHECKSUM_SHA256="28eb8504a2a436a117821aaa399b8c37f5363bd9fcb2ff814f6cb1b6264da311"
+SOURCE_DIR="WakeUp-116afedea73ba9779d3d641028b65fdb2dc19012"
+
+ARCHITECTURES="x86 x86_gcc2 x86_64"
```

Comment:
Remove "x86". We only have two official Haiku architectures, gcc2 (32bit) and gcc5 (64bit). It builds on both and doesn't provide a library that may be useful for gcc5 (32bit).